### PR TITLE
feature/text

### PR DESCRIPTION
**Pull request by billyxs**

### CODE REVIEW

File: packages/text/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Contributing
+
+Open a PR at https://bitbucket.org/hixme/hixme-ui.
```
Comment (seesleestak) [inline packages/text/CONTRIBUTING.md]: Maybe change to reference the github repo

File: packages/text/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+// paragraph warning
+<Text p warning>Are you sure?</Text>
+<Paragraph warning>Are you sure?</Paragraph>
+```
+
+## Props
+
+| Category        | Name            | Type        |
+---------------------------------------------------
```
Comment (seesleestak) [inline packages/text/README.md]: Adding pipes on this line should fix the table formatting for github

File: packages/text/CONTRIBUTING.md (hunk 1 above)
Comment (billyxs) [reply packages/text/CONTRIBUTING.md]: done!

File: packages/text/README.md (hunk 2 above)
Comment (billyxs) [reply packages/text/README.md]: thanks! done!

**Pull request merged**